Source/MaestroAverage.cpp

```diff
@@ -55,14 +55,29 @@ void Maestro::Average(const Vector<MultiFab>& phi, BaseState<Real>& phibar,
 
                 const Array4<const Real> phi_arr = phi[lev].array(mfi, comp);
 
-                AMREX_PARALLEL_FOR_3D(tilebox, i, j, k, {
-                    int r = AMREX_SPACEDIM == 2 ? j : k;
-#if (AMREX_SPACEDIM == 2)
-                    if (k == 0)
```

Comment:
Do we still need the `if (k==0)` ?

Reply:
For 2d problems `k` should always be 0 here